gallery/src/components/NxDrawer/NxDrawerExample.tsx

```diff
@@ -0,0 +1,253 @@
+              <NxButton variant="primary">Footer</NxButton>
+            </NxButtonBar>
+          </NxFooter>
+        </NxDrawer>
+      )}
+      {showDrawerWithFooterScroll && (
+        <NxDrawer id="nx-drawer-with-footer-scroll"
+                  onCancel={() => setShowDrawerWithFooterScroll(false)}
+                  aria-label="Drawer with Footer and Scroll">
+          <NxDrawer.Header>
+            <NxDrawer.Header.Title>A Simple Drawer With Footer</NxDrawer.Header.Title>
+          </NxDrawer.Header>
+          {content}
+          <NxFooter>
+            <NxP>
+              <em>Pastry</em> tiramisu jujubes biscuit marshmallow donut dragée.
+              Jelly beans chocolate bar jelly beans soufflé candy wafer topping.
+            </NxP>
+            <NxButtonBar>
+              <NxButton variant="primary">Footer</NxButton>
+            </NxButtonBar>
+          </NxFooter>
+        </NxDrawer>
+      )}
+      {showDrawerTruncated && (
+        <NxDrawer id="nx-drawer-simple-truncated"
+                  onCancel={() => setShowDrawerTruncated(false)}
+                  aria-label="Simple Drawer with Truncated Title">
+          <NxDrawer.Header>
+            <NxDrawer.Header.Title>A simple drawer with a very long title that is truncated</NxDrawer.Header.Title>
+          </NxDrawer.Header>
+          <NxDrawer.Content>
+            <NxP>
+              Powder tiramisu gingerbread I love gummi bears I love. Lollipop gingerbread bonbon chupa chups cookie
+              I love dessert cake. <strong>Pie</strong> candy canes liquorice jelly beans sweet roll. Jelly candy donut
+              cotton candy halvah. Fruitcake halvah I love cheesecake I love I love. Wafer sweet sweet roll apple
+              pie jelly-o cheesecake candy I love.
+            </NxP>
+          </NxDrawer.Content>
+        </NxDrawer>
+      )}
+      <NxPageMain>
+        <NxPageTitle>
+          <NxH1>Examples of Simple Drawers</NxH1>
+        </NxPageTitle>
+
+        <NxP>
+          Dragée pastry soufflé shortbread donut fruitcake. Ice cream tart bear claw I love
+          cotton candy marzipan cotton candy cake danish. Pie gingerbread marshmallow bear claw
+          halvah tiramisu cotton candy icing topping. Liquorice chupa chups dessert carrot cake
+          macaroon wafer. Marshmallow apple pie danish muffin cupcake icing dessert I love lemon
+          drops. Cupcake I <em>love</em> candy canes dragée croissant cookie chocolate muffin. Marshmallow
+          powder lollipop cotton candy bonbon lollipop liquorice chupa chups jelly-o. Biscuit I
+          love marzipan pastry pie ice cream chocolate bar dessert sweet. Cake topping cookie
+          chocolate pie cupcake. I love pastry donut croissant macaroon chocolate cake icing macaroon marshmallow.
+        </NxP>
+        <NxP>
+          <NxButton id="nx-drawer-simple-open-button" onClick={() => setShowDrawer(true)}>Open Simple Drawer</NxButton>
+          <br/>
+          <br/>
```

Comment:
I'd rather we just put each button in its own paragraph than use break tags for formatting.

Reply:
> I'd rather we just put each button in its own paragraph than use break tags for formatting.

Fixed here: 20c89142ec549c8d350974eb354923b61cc55971